optimade/validator/validator.py

```diff
@@ -39,7 +39,8 @@
         'elements HAS "Na"',
         'elements HAS ANY "Na","Cl"',
         'elements HAS ALL "Na","Cl"',
-    ],
+    ]
```

Comment:
Should we not replace these with just `MANDATORY_FILTER_EXAMPLES`, perhaps adding these lines to the file of examples? Or just dismissing them, since/if we already have these included?